core/modules/entity/entity.query.inc

```diff
@@ -947,7 +947,7 @@ class EntityFieldQuery {
       return $select_query->countQuery()->execute()->fetchField();
     }
     $return = array();
-    $this->orderedResults = array();
+    $this->ordered_results = array();
```

Comment:
Out of curiosity: why switching from camelCase to underscore?

Shouldn't we use camelCase for properties and methods, wherever possible, and underscores only, when inevitable for BC reasons (like classes that get extended in contrib...)?

Reply:
Agreed. Plus we can't change a public method without a very good reason to break compatibility.